testing/tests/hashtag-test.md

```diff
@@ -1,9 +1,10 @@
 Rust code that includes a "`#`" should be tested by skeptic without error.
 
 ```rust
+#[derive(derive_more::From)]
 struct Person<'a>(&'a str);
 fn main() {
-  let _ = Person("#bors");
+  let _ = Person::from("#bors");
```

Comment:
Is this necessary to test the behavior on Windows?

Reply:
This line isn't necessary. It's even enough to make sure that any external proc macro is visible, so in case of this PR, `use derive_more::From;`

Reply:
Okay, if we need the extra dependency I'm fine leaving it like this.

Can you fix up the formatting? I'll merge after that.